src/parameterizations/vertical/MOM_diabatic_driver.F90

```diff
@@ -251,6 +261,18 @@ module MOM_diabatic_driver
   type(time_type), pointer :: Time !< Pointer to model time (needed for sponges)
 end type diabatic_CS
 
+!> This structure has memory for used in calculating diagnostics of diffusivity
+type bflx_diags
+  real, pointer, dimension(:,:,:) :: &
+    Kd_int => NULL(), &  !< diapycnal diffusivity of interfaces [H Z T-1 ~> m2 s-1 or kg m-1 s-1]
+    N2_salt => NULL(), & !< Salinity contribution to squared buoyancy frequency at interfaces [T-2 ~> s-2]
+    N2_temp => NULL(), & !< Temperature contribution to squared buoyancy frequency at interfaces [T-2 ~> s-2]
+    Bflx_salt => NULL(), & !< Salinity contribution to buoyancy flux at interfaces [H2 T-3 ~> m2 s-3]
```

Comment:
More appropriate units for Bflux would probably be `[H Z T-3 ~> m2 s-3 or kg m-1 s-3]` (noting that [kg m-1 s-3] = [W m-3]) rather than the units of `[H2 T-3 ~> m2 s-3 or kg2 m-4 s-3]` as described here.

Reply:
This has been updated as suggested.